shoes-core/lib/shoes/color/dsl_helpers.rb

```diff
@@ -81,7 +83,11 @@ def image_file?(arg)
       end
 
       def image_pattern(path)
+        path = absolute_file_path(path)
         Shoes::ImagePattern.new path if File.exist?(path)
+      rescue FileNotFoundError => e
+        Shoes.logger.error(e.message)
+        return nil
```

Comment:
Just seemed a bit out of place, but yeah I can imagine... all good will merge away :)